src/exposurelog/routers/find_exposures.py

```diff
@@ -67,17 +75,40 @@ async def find_exposures(
     ),
     min_date: typing.Optional[datetime.datetime] = fastapi.Query(
         default=None,
-        description="Minimum date during the time the exposure was taken, *exclusive*; "
-        "TAI as an ISO string with no timezone information",
+        description="Minimum date during the time the exposure was taken, exclusive "
+        "(because that is how daf_butler performs a timespan overlap search). "
+        "TAI as an ISO string with no timezone information. "
+        "The date and time portions may be separated with a space or a T.",
     ),
     max_date: typing.Optional[datetime.datetime] = fastapi.Query(
         default=None,
-        description="Maximum date during the time the exposure was taken, exclusive; "
-        "TAI as an ISO string with no timezone information",
+        description="Maximum date during the time the exposure was taken, inclusive "
+        "(because that is how daf_butler performs a timespan overlap search). "
```

Comment:
Maybe just say "Butler" rather than "daf_butler"?

Reply:
I'm not entirely happy with "Butler" because it gives no context. (This service is more about log messages and less about daf_butler) but lsst.daf.butler.Butler is also not entirely correct, and in any case it is the Registry that is doing the searching. Maybe I'll use daf_butler Registry.